crates/qasm3/Cargo.toml

```diff
@@ -18,4 +18,4 @@ extension-module = ["pyo3/extension-module"]
 pyo3.workspace = true
 indexmap.workspace = true
 hashbrown.workspace = true
-oq3_semantics = "0.0.7"
+oq3_semantics = { git = "https://github.com/jlapeyre/openqasm3_parser", branch = "update-for-qiskit-importer"}
```

Comment:
Is there a release on crates.io that we should be using for this? I don't think we want to pull in a branch from git to build the parser?

Reply:
It was my intention to depend on a released version before merging this PR.

Reply:
I opened this PR on March 26. I tagged and released parser v 0.5.0 on March 25. I'm pretty sure that was the intended version. I'll check.

Reply:
I published oq3_semantics (and all crates) 0.6.0. And edited the Qiskit Cargo.toml to depend on it. fd2c6d5